src/test/java/com/flightstats/hub/spoke/FileSpokeStoreTest.java

```diff
@@ -20,32 +20,39 @@
 import java.util.Collection;
 import java.util.List;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
 
+@Slf4j
 public class FileSpokeStoreTest {
+
+    private static final SpokeProperty spokeProperty = new SpokeProperty(PropertyLoader.getInstance());
+    private static final int ttlMinutes = spokeProperty.getTtlMinutes(SpokeStore.WRITE);
     public static final byte[] BYTES = new byte[]{0, 2, 3, 4, 5, 6};
-    private final static Logger logger = LoggerFactory.getLogger(FileSpokeStoreTest.class);
-    private String tempDir;
+
     private FileSpokeStore spokeStore;
-    private static final int ttlMinutes = HubProperties.getSpokeTtlMinutes(SpokeStore.WRITE);
+    private String tempDir;
 
     @Before
-    public void setUp() throws Exception {
+    public void setUp() {
         tempDir = Files.createTempDir().getPath();
-        HubProperties.setProperty("spoke.write.path", tempDir);
+        PropertyLoader.getInstance().setProperty("spoke.write.path", tempDir);
```

Comment:
Might be another good candidate for mocking so we can avoid mutating test-suite-wide state...